pkg/neorpc/filters.go

```diff
@@ -49,6 +53,12 @@ type (
 	}
 )
 
+// SubscriptionFilter is an interface for all subscription filters.
+// IsValid checks whether the filter is valid.
```

Comment:
IsValid checks whether the filter is valid and returns a specific [ErrInvalidSubscriptionFilter] error if not.